src/components/commentPopup.css

```diff
@@ -0,0 +1,41 @@
+.popup {
```

Comment:
- [ ] **OPTIONAL**
It'd be great if you shrank the size of the popup here and centered it 😄